examples/agentInteractionsDemo/app.js

```diff
@@ -175,13 +175,53 @@ new Vue({
         const usersApi = new platformClient.UsersApi();
         const qualityApi = new platformClient.QualityApi();
         const conversationsApi = new platformClient.ConversationsApi();
+        const externalContactsApi = new platformClient.ExternalContactsApi();
         Vue.prototype.$usersApi = usersApi;
         Vue.prototype.$qualityApi = qualityApi;
         Vue.prototype.$conversationsApi = conversationsApi;
+        Vue.prototype.$externalContactsApi = externalContactsApi;
 
         let authenticated = false;
         let agentUserId = "";
 
+        function getCustomerParticipant(conv) {
```

Comment:
So it's kind of a nit since this is a demo app, but I'll throw it out there anyway. Generally, we like to keep functions pure, highlighting any mutation or side effects with an explicit function name. It can quickly become unclear where mutations occur if we don't highlight them.

In this case, the conversation is passed in and we're mutating the attached customer participant. I think we should either a) change the function name, b) isolate the mutation and make it more obvious, or c) avoid the mutation altogether.

In an ideal world, I think we'd avoid this mutation altogether, but given that we were already doing this I think `b` is the best. I think a better way to delegate the responsibilities here could be something like:

```
const customer = getCustomerParticipant(conversation);
customer.name = await getCustomerName(customer);
```

^ Then it's left up to the consumers to decide whether or not the conversation should be mutated.